bin/fin

```diff
@@ -919,7 +938,7 @@ check_project_unique ()
 _confirm ()
 {
 	# Skip checks if not running interactively (not a tty or not on Windows)
-	if ! is_tty; then return 0; fi
+	if ! is_tty || is_travis; then return 0; fi
```

Comment:
I think we should add a global `-y` flag to skip confirmations in `_confirm`.

Travis is not the only CI system that's using/faking interactive sessions and others may run into the need to skip confirmations.

Alternatively, commands can be run in a sub-shell in Travis to force a non-interactive environment, e.g. `run (fin init)` (I have not tested this, though).

Reply:
Good notice. I'd like to keep "supporting all possible generic CIs" separately though. https://github.com/docksal/docksal/issues/707